backtrace-library/src/main/java/backtraceio/library/models/BacktraceNativeData.java

```diff
@@ -0,0 +1,71 @@
+package backtraceio.library.models;
+
+import android.content.Context;
+import android.icu.util.Output;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import backtraceio.library.common.FileHelper;
+import backtraceio.library.common.MultiFormRequestHelper;
+import backtraceio.library.logger.BacktraceLogger;
+import backtraceio.library.models.json.BacktraceReport;
+
+public class BacktraceNativeData {
+    private String minidumpPath;
+    private List<String> attachments;
```

Comment:
can be final

Reply:
Can't be final as it has to be modified.